Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml

```diff
@@ -0,0 +1,888 @@
+id: Cloud IAM User Access Investigation
+version: -1
+contentitemexportablefields:
+  contentitemfields: {}
+name: Cloud IAM User Access Investigation
+description: |-
+  Investigate and respond to Cortex XSIAM alerts where a Cloud IAM user access key is used suspiciously to access the cloud environment.
+  The following alerts are supported for AWS, Azure, and GCP environments.
+  Penetration testing tool attempt
+  Penetration testing tool activity
+  Suspicious API call from a Tor exit node
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 40b4c735-392d-4c86-8ab3-6d49441a813d
+    type: start
+    task:
+      id: 40b4c735-392d-4c86-8ab3-6d49441a813d
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "61"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 550,
+          "y": -190
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: c48a2862-f3e1-4dd3-89b7-8590f645298b
+    type: title
+    task:
+      id: c48a2862-f3e1-4dd3-89b7-8590f645298b
+      version: -1
+      name: 'Remediation '
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "81"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 550,
+          "y": 1250
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "19":
+    id: "19"
+    taskid: 1e498a1e-df55-4805-8fa9-973ba6554ac1
+    type: title
+    task:
+      id: 1e498a1e-df55-4805-8fa9-973ba6554ac1
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 510,
+          "y": 1910
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "21":
+    id: "21"
+    taskid: 08cf3974-ddc8-4694-8a1a-e737b3e69973
+    type: condition
+    task:
+      id: 08cf3974-ddc8-4694-8a1a-e737b3e69973
+      version: -1
+      name: Manual decision making - true/false-positive alert
```

Comment:
Maybe "Manual verdict decision" ? or something like that .shorter

Reply:
Done